bin/webpack-configs.js

```diff
@@ -982,6 +982,125 @@ const getInteractivityAPIConfig = ( options = {} ) => {
 	};
 };
 
+/**
+ * Build config for Interactivity Blocks in the frontend context.
+ *
+ * @param {Object} options Build options.
+ */
+const getInteractivityBlocksConfig = ( options = {} ) => {
+	let { fileSuffix } = options;
+	const { alias, resolvePlugins = [] } = options;
+	fileSuffix = fileSuffix ? `-${ fileSuffix }` : '';
+	const resolve = alias
+		? {
+				alias,
+				plugins: resolvePlugins,
+		  }
+		: {
+				plugins: resolvePlugins,
+		  };
```

Comment:
I've observed that these lines of code are being duplicated. It might be a good idea to consolidate them into a reusable function. I believe there's room for improvement to achieve a more DRY code structure in this file. What are your thoughts on this?